src/GraphODataTemplateWriter/.config/JavaSettings.json

```diff
@@ -119,14 +119,14 @@
           { "Template": "IBaseEntityCollectionReferenceRequest", "SubProcessor": "CollectionReferenceProperty",                               "Name": "IBase<PropertyType>CollectionReferenceRequest", "Exclude": "nextExpectedRanges;roles", "Ignore": "ignorableCollection" },
           { "Template": "BaseEntityCollectionReferenceRequest", "SubProcessor": "CollectionReferenceProperty",                                "Name": "Base<PropertyType>CollectionReferenceRequest", "Exclude": "nextExpectedRanges;roles", "Ignore": "ignorableCollection" },
 
-          { "Template": "IEntityRequest", "SubProcessor": "StreamProperty",                                                                   "Name": "I<Class><PropertyType>Request", "Include": "mainLogo;thumbnailPhoto;content" },
-          { "Template": "EntityRequest", "SubProcessor": "StreamProperty",                                                                    "Name": "<Class><PropertyType>Request", "Include": "mainLogo;thumbnailPhoto;content" },
-          { "Template": "IEntityRequestBuilder", "SubProcessor": "StreamProperty",                                                            "Name": "I<Class><PropertyType>RequestBuilder", "Include": "mainLogo;thumbnailPhoto;content" },
-          { "Template": "EntityRequestBuilder", "SubProcessor": "StreamProperty",                                                             "Name": "<Class><PropertyType>RequestBuilder", "Include": "mainLogo;thumbnailPhoto;content" },
-          { "Template": "IBaseStreamRequest", "SubProcessor": "StreamProperty",                                                               "Name": "IBase<Class><PropertyType>Request", "Include": "mainLogo;thumbnailPhoto;content" },
-          { "Template": "BaseStreamRequest", "SubProcessor": "StreamProperty",                                                                "Name": "Base<Class><PropertyType>Request", "Include": "mainLogo;thumbnailPhoto;content" },
-          { "Template": "IBaseEntityRequestBuilder", "SubProcessor": "StreamProperty",                                                        "Name": "IBase<Class><PropertyType>RequestBuilder", "Include": "mainLogo;thumbnailPhoto;content" },
-          { "Template": "BaseEntityRequestBuilder", "SubProcessor": "StreamProperty",                                                         "Name": "Base<Class><PropertyType>RequestBuilder", "Include": "mainLogo;thumbnailPhoto;content" },
+          { "Template": "IEntityRequest", "SubProcessor": "StreamProperty",                                                                   "Name": "I<Class><PropertyType>Request", "Include": "mainLogo;thumbnailPhoto;content;logo" },
```

Comment:
We should probably come up with a solution to either generate this list programmatically from Stream properties, or at least some mechanism to notify us when there is a new stream property in the metadata so that we can update this list.

Reply:
hopefully the build will end up failing if it's not there.

Reply:
C# doesn't use the Include list and just assumes that all stream properties are accessed the same way. I don't know if that is correct, but it hasn't caused us problems, yet. 

The build will fail if not set for Java.